pkg/repo/create.go

```diff
@@ -0,0 +1,76 @@
+package repo
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/repo/badgerdb"
+	"go.expect.digital/translate/pkg/repo/mysql"
+)
+
+type RepoDB int
+
+const (
+	MySQL RepoDB = iota
+	BadgerDB
+)
+
+var RepoDBNames = map[RepoDB]string{
+	MySQL:    "mysql",
+	BadgerDB: "badgerdb",
+}
+
+func (r RepoDB) String() string {
+	return RepoDBNames[r]
+}
+
+// SupportedDBs returns a list of supported databases.
+func SupportedDBs() []string {
+	dbs := make([]string, 0, len(RepoDBNames))
+	for _, v := range RepoDBNames {
+		dbs = append(dbs, v)
+	}
+
+	return dbs
+}
```

Comment:
Why not simply have a list of supported databases? It simplifies the whole thing.

```suggestion
var supportedDBs = [...]string{"mysql", "badgerdb"}
```

Reply:
i guess i had something in mind yesterday, but yeah looking now it looks kinda pointless